nixos/doc/manual/release-notes/rl-2511.section.md

```diff
@@ -210,6 +208,8 @@
 
 - `netbox-manage` script created by the `netbox` module no longer uses `sudo -u netbox` internally. It can be run as root and will change it's user to `netbox` using `runuser`
 
+- `emacs` now disables the GC mark trace buffer by default. This improves GC performance by 5%, but can make GC issues harder to debug. This is configurable with `withGcMarkTrace`.
```

Comment:
I think this release note should be added [here][1].

[1]: https://github.com/NixOS/nixpkgs/blob/master/doc/release-notes/rl-2511.section.md#other-notable-changes-sec-nixpkgs-release-2511-notable-changes